bindings/jupyter/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: develop install clean build publish-test
+
+develop:
+	python setup.py develop
+
+install:
+	python setup.py install
+
+clean:
+	rm -rf dist
+
+build:
+	rm -rf dist
+	python setup.py sdist bdist_wheel
+
+publish-test:
```

Comment:
Do we need another `publish` - to the non-Test PyPI for `pip install`?

Reply:
yup that's true